### ISSUE TITLE
[content/question] new Republic ship; the "Destroyer"

### ISSUE THREAD

**Issue by KiLEdEnNis** [endless-sky/endless-sky#1456]
I haven't finished my first ship (my own project) yet but it's because it's a totally different style, and for some ship-creating experience it'd be a better idea for me to create a human ship, which pairs with something that I'd like to see; a new RNS ship; the Destroyer (between Frigate and Cruiser; firepower that may rival the Cruiser, but less survivability, smaller etc.)

is there "room" to make a new Republic ship like that? would it be "appreciated" or is it undeeded? (also because maybe some missions would require them to be added, the missions need new balance testing, new fleet variants etc.)
I'd be willing to test each story with a ~15M ship if it would mean that I could try and create a new ship (and if I have trouble with it, which I doubt, could someone help a bit?)

if I need help or anything I can ask on the google group

btw, a side question (to MZ); weren't you planning to create a "Battleship" for the Republic as well?

**Comment by jafdy** [comment]
You should talk to @LocalGod79, he's creating a Republic storyline. It may fit in there.

**Comment by KiLEdEnNis** [comment]
@jafdy can you give a link or something to information about what he is making? (if there is one)

**Comment by jafdy** [comment]
In the google group, other than that, no.

**Comment by Wrzlprnft** [comment]
At the moment, due to his family (close realtive died recently and it is summer so... Children entertainment) he is quite often offline at the moment. Just message him via google mail, or steam, maybe he gives you any specific information

**Comment by KiLEdEnNis** [comment]
@Wrzlprnft rip :(
also thanks both :)

**Comment by Wrzlprnft** [comment]
Otherwise: edit is not possible on phone:  a better warship similar to the dreadnought development is quite likely, but "megaships" may have to wait till further human <-> alien relationships

**Comment by KiLEdEnNis** [comment]
@Wrzlprnft Destroyer would be about Cruiser in firepower but inferior in other stats (a faster weaker version)
and w/ the Battleship I don't mean a "megaship", just an extremely rare ship that's better than the Cruiser/Carrier (barely/no bays, a slow but strong capital ship)
because the Navy Admirals use plain Cruisers, there should be a Battleship for them (which you first encounter end of the FW story, maybe when the Navy cuts the FW off, with a new capital ship, which is that Battleship)
so in short, the Cruiser is a tier above the Frigate, the Battleship a tier above the Cruiser

**Comment by Wrzlprnft** [comment]
I just wanted to state the general plan. The navy, due to havin superior infrastructure, even after the bombing, may develop 2 instead of one extra warship, but i still say: let's see how it works out.

**Comment by LocalGod79** [comment]
i don't believe that another warship is required.  the Navy lineup has six warships in it already - more ships than MegaParsec produces in total...

either the Cruiser or the Carrier fills the "Battleship" role quite well (even without fighters/drones), and a Frigate and Gunboat or Rainmaker fleet would fill the proposed "Destroyer" niche.

by definition "(Torpedo-Boat/Submarine) Destroyer" replaced "Frigate" in U.S. Navy nomenclature, as they fill virtually the same roles.

**Comment by KiLEdEnNis** [comment]
@LocalGod79 I know the Navy doesn't need it, but I thought it'd be a nice addition to have a bit more versatility, and then create some ships for the other corporations
and what do you mean the Navy has 6 warships? unless you mean the Lance as well, they have the Rainmaker, Gunboat, Frigate, Cruiser and Carrier

**Comment by LocalGod79** [comment]
i think the point i was trying to make was: of all the shipyards already implemented, the Navy's would probably benefit the *least* from additional ships - especially since you'd have some pretty significant workload testing for the FW plot once it was added to fleets...

**Comment by KiLEdEnNis** [comment]
I know the combat drone is Navy, but it's not exactly a warship
and too bad about the new ship then D:
maybe I'll just make a Merchant Warship

**Comment by jafdy** [comment]
The FW carriers already created that work. On the whole adding even more new, unbuyable ships would not make the game any better. It is more likely to send the Steam crowd in to hysterics over a ship they can't get.

**Comment by KiLEdEnNis** [comment]
they can be hijacked but I get the point, if we would even make a new RNS ship we'd have to wait until the Navy story is done and they CAN get them

**Comment by endless-sky** [comment]
Right now the biggest content-creation need in the game is for new missions (including entire story lines) and new alien species (to fill in all that empty space in the galaxy). New human ships are a rather low priority because they already have so much more variety of ships than every other species out there. So if you're interested in ship creation, I'd recommend trying to create a new species rather than just adding new human ships.

**Comment by KiLEdEnNis** [comment]
@endless-sky I do have a project (early stages, and making a smaller pre-story for multiple reasons)
on the google group, one of the top posts "feedback for the first ship in my project" shows the 1st ship (immediately the largest one)
I can PM both the pre-story story and main story ideas on google groups if you like (to get feedback and change things before I make them, all the pre-story story needs is planet descriptions, names, more detailed alienish human hails etc. basically a few more tweaks, apart from the story, I pasted the pre-story map coordinates on the google group as well)
